### ISSUE TITLE
Unhandled Rejection! This is a bug and should be reported [FetchError]

### ISSUE THREAD

**Issue by enavermate** [raineorshine/npm-check-updates#1521]
## Steps to Reproduce

.ncurc:

```js
{
  "reject": [
    "react",
    "react-native",
    "@types/react",
    "@react-navigation/bottom-tabs",
    "@react-navigation/native",
    "@react-navigation/native-stack",
    "@react-navigation/stack",
    "@react-native/babel-preset",
    "@react-native/metro-config",
    "eslint-plugin-react-native",
    "react-native-image-picker",
    "react-test-renderer",
    "@react-native-community/cli",
    "@react-native-community/cli-platform-android",
    "@react-native-community/cli-platform-ios",
    "metro",
    "metro-minify-terser",
    "react-native-modal",
    "react-native-maps",
    "react-native-draggable-flatlist",
    "@react-native-firebase/app",
    "@react-native-firebase/auth",
    "@react-native-firebase/messaging"
  ]
}

```

Dependencies:

```json

```

Steps:

`ncu -i` - 2 times

## Current Behavior

First launch and output error: 

```
ncu -i
[=============-------] 48/72 66%e [FetchError]: request to https://registry.npmjs.org/%40stripe%2Fstripe-react-native failed, reason: read ECONNRESET
    at ClientRequest.<anonymous> (<user>/.nvm/versions/node/v20.18.3/lib/node_modules/npm-check-updates/build/index.js:360:118368)
    at ClientRequest.emit (node:events:518:28)
    at emitErrorEvent (node:_http_client:101:11)
    at TLSSocket.socketErrorListener (node:_http_client:504:5)
    at TLSSocket.emit (node:events:518:28)
    at emitErrorNT (node:internal/streams/destroy:169:8)
    at emitErrorCloseNT (node:internal/streams/destroy:128:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  code: 'ECONNRESET',
  errno: 'ECONNRESET',
  syscall: 'read',
  type: 'system'
}
[==============------] 51/72 70%Unhandled Rejection! This is a bug and should be reported: https://github.com/raineorshine/npm-check-updates/issues
```

Second launch and output error: 

```
ncu -i
Using config file /<project>/.ncurc.json
Using yarn
Upgrading /<project>/package.json
[=-------------------] 3/72 4%n: 504 Gateway Timeout - GET https://registry.npmjs.org/%40miblanchard%2Freact-native-slider
    at /<user>/.nvm/versions/node/v20.18.3/lib/node_modules/npm-check-updates/build/index.js:365:951
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  headers: [Object: null prototype] {
    date: [ 'Thu, 12 Jun 2025 18:14:02 GMT' ],
    'content-type': [ 'text/html' ],
    'transfer-encoding': [ 'chunked' ],
    connection: [ 'keep-alive' ],
    'cf-ray': [ '94eb52da5c250110-GRU' ],
    'cf-cache-status': [ 'DYNAMIC' ],
    vary: [ 'Accept-Encoding' ],
    'set-cookie': [
      '_cfuvid=OxgLXIgHyLCF4opAuRs3xm6OmzOBzbibIYhX7JcFnns-1749752042702-0.0.1.1-604800000; path=/; domain=.npmjs.org; HttpOnly; Secure; SameSite=None'
    ],
    server: [ 'cloudflare' ],
    'x-fetch-attempts': [ '1' ]
  },
  statusCode: 504,
  code: 'E504',
  method: 'GET',
  uri: 'https://registry.npmjs.org/%40miblanchard%2Freact-native-slider',
  body: <Buffer 3c 68 74 6d 6c 3e 0d 0a 3c 68 65 61 64 3e 3c 74 69 74 6c 65 3e 35 30 34 20 47 61 74 65 77 61 79 20 54 69 6d 65 2d 6f 75 74 3c 2f 74 69 74 6c 65 3e 3c ... 82 more bytes>,
  pkgid: '@miblanchard/react-native-slider'
}
[=-------------------] 5/72 6%Unhandled Rejection! This is a bug and should be reported: https://github.com/raineorshine/npm-check-updates/issues
```

**Comment by dietler** [comment]
I'm getting this error today as well:

`[==================--] 16/18 88%n: 504 Gateway Timeout - GET https://registry.npmjs.org/%40vitest%2Fui
    at C:\Users\AndyI\AppData\Local\Temp\bunx-847564299-npm-check-updates@latest\node_modules\npm-check-updates\build\index.js:365:951
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5) {
  headers: [Object: null prototype] {
    date: [ 'Thu, 12 Jun 2025 19:24:28 GMT' ],
    'content-type': [ 'text/html' ],
    'transfer-encoding': [ 'chunked' ],
    connection: [ 'keep-alive' ],
    'cf-ray': [ '94ebba24cb83520e-LAX' ],
    'cf-cache-status': [ 'DYNAMIC' ],
    vary: [ 'Accept-Encoding' ],
    'set-cookie': [
      '_cfuvid=OwTABsi.3LMCAOlyUdDRxqZsMKklpcifx9PmRI_ZlkM-1749756268008-0.0.1.1-604800000; path=/; domain=.npmjs.org; HttpOnly; Secure; SameSite=None'
    ],
    server: [ 'cloudflare' ],
    'x-fetch-attempts': [ '1' ]
  },
  statusCode: 504,
  code: 'E504',
  method: 'GET',
  uri: 'https://registry.npmjs.org/%40vitest%2Fui',
  body: <Buffer 3c 68 74 6d 6c 3e 0d 0a 3c 68 65 61 64 3e 3c 74 69 74 6c 65 3e 35 30 34 20 47 61 74 65 77 61 79 20 54 69 6d 65 2d 6f 75 74 3c 2f 74 69 74 6c 65 3e 3c ... 82 more bytes>,
  pkgid: '@vitest/ui'
}
[===================-] 17/18 94%Unhandled Rejection! This is a bug and should be reported: https://github.com/raineorshine/npm-check-updates/issues
error: script "check for npm updates" exited with code 1`

**Comment by raineorshine** [comment]
Hi, thanks for reporting.

This appears to be a registry server error. Is this still occurring for you? What do you get with `npx pacote manifest react`?

**Comment by fsrv-xyz** [comment]
That seemed to be related to the npm outage announced via their status page. I experienced the same errors on multiple regions which are resolved now due to the resolved state of the outage.

See https://status.npmjs.org/incidents/dn5mcp85737y for further details.
